pkg/printers/internalversion/printers_test.go

```diff
@@ -829,29 +829,19 @@ func TestPrintNodeStatus(t *testing.T) {
 			node: api.Node{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:   "foo10",
-					Labels: map[string]string{"node-role.kubernetes.io/master": "", "kubernetes.io/role": "node", "kubeadm.alpha.kubernetes.io/role": "node"},
+					Labels: map[string]string{"node-role.kubernetes.io/master": "", "node-role.kubernetes.io/proxy": "", "kubernetes.io/role": "node"},
 				},
 			},
-			status: "Unknown,Master",
+			status: "master,node,proxy",
```

Comment:
> shall we use strings.Fields(buffer.String())[2] instead of this

a tighter check could be ok, but I'd rather not make our tests brittle to adding or removing unrelated columns. follow up is fine